analyses/data/world_bank_hiv_prevalence.csv

```diff
@@ -0,0 +1,271 @@
+﻿"Data Source","World Development Indicators",
```

Comment:
This file isn't too large so it's not necessarily a problem to add it to the repo, but we should think about what data we want to add to the repo directly, vs what we want to store externally, or provide the script to download the data rather than the data itself (which in this case might be as simple as e.g. `sc.download('https://api.worldbank.org/v2/en/indicator/SH.DYN.AIDS.ZS?downloadformat=csv')`).